modules/gatsby/src/components/layout/table-of-contents.jsx

```diff
@@ -196,11 +197,11 @@ const SafeLink = ({
 
   return (
     <TocEntry $depth={depth} title={name} onClick={() => toggleEntry(id)}>
-      {hasChildren && <TocChevron $depth={depth} />}
+      {hasChildren && <TocChevron $depth={depth} $isTocOpen={isTocOpen} />}
```

Comment:
i see what you did there, I didn't implement rotating the toc chevron because design had it facing down whether the entry was open or not